pkg/middleware/middleware.go

```diff
@@ -79,24 +74,153 @@ func AuthenticationMiddleware() gin.HandlerFunc {
 			return
 		}
 
-		userId := int64(claims["user"].(map[string]interface{})["id"].(float64))
+		if unverifiedClaims.Issuer != "" {
```

Comment:
This case is valid for tokens issued by the application? If yes, we can add issuer for the JWT tokens created by the application instead.